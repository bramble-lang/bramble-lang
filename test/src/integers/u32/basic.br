fn my_main() -> i64 {
    let x:u32 := 4u32;
    let y:u32 := test();
    let z:u32 := test2(640000u32);
    let z2:u32 := test2(2147483647u32);
    let z3:u32 := test2(4294967295u32);

    project::std::io::writeu32ln(x);
    project::std::io::writeu32ln(y);
    project::std::io::writeu32ln(z);
    project::std::io::writeu32ln(z2);
    project::std::io::writeu32ln(z3);
    
    return 0;
}

fn test() -> u32 {
    return 5u32;
}

fn test2(i: u32) -> u32 {
    return i;
}