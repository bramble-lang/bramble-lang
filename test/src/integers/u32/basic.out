4
5
640000
2147483647
4294967295
